tempto-core/src/main/java/io/prestodb/tempto/fulfillment/table/hive/HiveTableDefinition.java

```diff
@@ -46,7 +46,7 @@
     private HiveTableDefinition(TableHandle handle, String createTableDDLTemplate, Optional<HiveDataSource> dataSource, Optional<List<PartitionDefinition>> partitionDefinitions)
     {
         super(handle);
-        checkArgument(dataSource.isPresent() ^ partitionDefinitions.isPresent(), "either dataSource or partitionDefinitions must be set");
+        checkArgument(dataSource.isPresent() != partitionDefinitions.isPresent(), "either dataSource or partitionDefinitions must be set (but not both)");
```

Comment:
what is the difference?

Reply:
Semantically, none.

Reply:
So what is the motivation, and why does the commit begin with `Fix`?